lib/langchain/llm/openai.rb

```diff
@@ -229,5 +232,10 @@ def with_api_error_handling
     def validate_max_tokens(messages, model)
       LENGTH_VALIDATOR.validate_max_tokens!(messages, model)
     end
+
+    def extract_response(response)
+      results = response.dig("choices").map { |choice| choice.dig("message", "content") }
+      (results.size == 1) ? results.first : results
```

Comment:
I think it's a little odd that this method returns either a string or an array (different data structures). But maybe we can change this when we introduce `Langchain::LLM::Response`

Reply:
Yeah, that's what I was thinking